### PR TITLE
Feature / Integrate CMS on Product Page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Add PriceCalculator block.
- Add ProductSummary block.
- Extract Tabs and remove from ProductPage.
- Add StoryblokBlockName enum.
- Calculate 'extra props' based on ProductPage context.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #321** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/321" target="_blank">Graphite</a> 
    * **PR #323** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/323" target="_blank">Graphite</a> 
      * **PR #324** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/324" target="_blank">Graphite</a> 
        * **PR #325** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/325" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/325?utm_source=stack-comment).

File: apps/store/src/blocks/ProductSummary.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styled from '@emotion/styled'
+import { storyblokEditable } from '@storyblok/react'
+import { ProductSummary } from '@/components/ProductPage/ProductSummary/ProductSummary'
+import { SbBaseBlockProps } from '@/services/storyblok/storyblok'
+
+type Props = {
+  title: string
+  description: string
+  starScore: number
+  gradient: readonly [string, string]
```
Comment (guilhermespopolin) [inline apps/store/src/blocks/ProductSummary.tsx:10]: Just a question:
Why `readonly` in this case?

File: apps/store/src/components/ProductPage/ProductPageContext.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react'
```
Comment (guilhermespopolin) [inline apps/store/src/components/ProductPage/ProductPageContext.tsx:1]: Maybe we can remove this right?!

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/blocks/ProductSummary.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/blocks/ProductSummary.tsx:10]: I guess we technically want it to be readonly

But I added it as this is the type that matches:

```
const gradient = ["#000", "#bbb"] as const
```

File: apps/store/src/components/ProductPage/ProductPageContext.tsx (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/components/ProductPage/ProductPageContext.tsx:1]: oh whoops, yes it should be removed!

**Pull request merged**